src/main/java/uk/gov/companieshouse/api/accounts/service/impl/TnepValidationServiceImpl.java

```diff
@@ -55,19 +55,10 @@ public boolean validate(String ixbrl, String location) {
             Results results = validatIxbrlAgainstTnep(ixbrl, location);
 
             if (hasPassedTnepValidation(results)) {
-                addToLog(false, null, results, location,
-                    "Ixbrl is valid. It has passed the TNEP validation");
-
                 isIxbrlValid = true;
-
-            } else {
-                addToLog(true, null, results, location,
-                    "Ixbrl is invalid. It has failed the TNEP validation");
             }
 
         } catch (Exception e) {
-            addToLog(true, e, null, location,
```

Comment:
We shouldn't just be swallowing the exception, we should still log it

Reply:
I rolled back and removed only "results" from the **addToLog**